File: VB/XtraReport1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraReports.UI
' ...

Namespace PreviewLabelClick
	Partial Public Class XtraReport1
		Inherits DevExpress.XtraReports.UI.XtraReport
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub xrLabel1_PreviewClick(ByVal sender As Object, ByVal e As PreviewMouseEventArgs) Handles xrLabel1.PreviewClick
			MessageBox.Show(e.Brick.Text)
		End Sub

		Private Sub xrLabel2_PreviewDoubleClick(ByVal sender As Object, ByVal e As PreviewMouseEventArgs) Handles xrLabel2.PreviewDoubleClick
			MessageBox.Show(e.Brick.Text)
		End Sub

	End Class
End Namespace

File: CS/XtraReport1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraReports.UI;
// ...

namespace PreviewLabelClick {
    public partial class XtraReport1 : DevExpress.XtraReports.UI.XtraReport {
        public XtraReport1() {
            InitializeComponent();
        }

        private void xrLabel1_PreviewClick(object sender, PreviewMouseEventArgs e) {
            MessageBox.Show(e.Brick.Text);
        }

        private void xrLabel2_PreviewDoubleClick(object sender, PreviewMouseEventArgs e) {
            MessageBox.Show(e.Brick.Text);
        }

    }
}
